doc/cephfs/snapshots.rst

```diff
@@ -2,43 +2,45 @@
 CephFS Snapshots
 ================
 
-CephFS snapshots create an immutable view of the file system at the point
-in time they are taken. CephFS support snapshots which is managed in a 
-special hidden subdirectory named ``.snap`` .Snapshots are created using
-``mkdir`` inside this directory.
+CephFS snapshots create an immutable view of the file system at the point in
+time they are taken. CephFS snapshots are managed in a hidden subdirectory
```

Comment:
I would swap special for hidden, as it isn’t really hidden except by ls.  If it were hidden one couldn’t run mkdir